code/api/utility.py

```diff
@@ -0,0 +1,12 @@
+from django.http.response import HttpResponseBadRequest
+from django.core.paginator import Paginator
+
+# https://docs.djangoproject.com/en/3.2/topics/pagination/ - Pagination
+def getPaginated(data, page, size):
+    try:
+        if page < 1 or size < 1:
+            return HttpResponseBadRequest("Malformed query: page and size must be > 0")
+    except:
+        return HttpResponseBadRequest("Malformed query")
+    p = Paginator(data, size)
+    return p.page(page)
```

Comment:
Server crashes and returns a 500 status code if we request a page with no results
`django.core.paginator.EmptyPage: That page contains no results`

Can we add a simple try-except block to catch this error? Just so it doesn't return a 5XX code. I think it would make sense to just return an empty list but anything other than a server crash is probably good.

Reply:
Sounds good